book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -8,24 +8,324 @@ This can involve hosting different events that are attended by people with lived
+It is important to acknowledge how your choices may impact the experience and inclusion of people with lived experience at your event.
+
+(cl-inclusive-events-timing-considerations)=
+### Timeline and Timing of Event 
+After identifying the "why" and "who" for your event, it is important to plan "when" the event will take place, and to develop a timeline for when tasks need to be completed in order to run your event. 
+
+Organizers should aim to avoid scheduling events on national holidays or religious and cultural days to accommodate attendees of various backgrounds, and depending on the scale of the event, you may also want to consider avoiding other days and timings that would make it difficult to attend for your audience. 
+For example, events during school holidays or sessions that overlap with school pick-up and drop-off times may make it difficult for people with school-age children to attend. 
+
+You should seek to work with your specific community to understand what their needs may be. 
+Many members of the public and people with lived experience may engage in research projects as a volunteer and will find it difficult to get time off from their workplace to attend an event. 
+Similarly, researchers may find it difficult to attend sessions outside of working hours. 
+
+Although scheduling is difficult for any form of event organizing, please take the time to consider what barriers an event's timing will have on your ability to be inclusive of people with lived experience and refer to your local area for any specific dates to avoid, and you can view an interfaith events calendar here: https://www.interfaith-calendar.org/. 
+
+When planning the dates for the event, it is also useful to consider the potential travel times of your attendees. 
+If your event takes place in person, try to choose a date and start time that takes into account how long your attendees will need to travel. 
+For example, having your event start at 10:30 a.m. allows for inter-city travel the morning before the event. 
+
+(cl-inclusive-events-timeline-planning)=
+#### Creating a Planning Timeline
+Your planning timeline will vary greatly based on the scale of your event. 
+We have highlighted areas where your timeline should take into account any considerations that would be specifically supportive of people with lived experience.
+
+Although it is always necessary to allow for appropriate time to source speakers, people with lived experience may require more notice than is traditionally planned for in order to support their attendance and speak at an event. 
+People with lived experience may have additional constraints such as medical appointments, caring responsibilities, support and travel needs, and preparation time which can require significant time in order to plan for and implement. 
+Try to identify potential speakers as early as possible and book dates in their diaries that would allow them to make plans that would support their engagement in the event.
```

Comment:
Try to identify potential speakers as early as possible and book dates that facilitate their organization to participate in the event.
```suggestion
Try to identify potential speakers as early as possible and book dates in their diaries that would allow them to make plans that would support their engagement in the event.
```

Reply:
Hi @JFormoso. 

Are you suggestion that the speakers need to facilitate their employer/organization, or that they themselves need to get organized in order to take part in the event?